### ISSUE TITLE
Design Menüs bei nicht auffindbarer Seite

### ISSUE THREAD

**Issue by julian-weide** [pfadfinden/wordpress-theme-updater#17]
Hallo Leute, 
mir ist wieder eine Kleinigkeit aufgefallen. Wenn man auf einer nicht-existenten Seite landet, wird das Design des Hauptmenüs und des Footers nicht richtig geladen. 
Hier ein Beispiel: http://konradstein.de/beliebige-nicht-auffindbare-seite
Schöne Woche
Julian

**Comment by JoshuaWolff** [comment]
Hier für muss die 404.php bearbeitet werden.

In dem Fall, wo ein Child-Theme genutzt wird kann man einfach die "404.php" bearbeitetn und den Code aus der Index.php oder eines andere Page-templates koppieren.

Wenn du kein Child-Theme hast, musst du darauf warten, dass die "404.php" im Template ersetzt bzw. bearbeitet wird.

**Comment by larsboehnke** [comment]
Danke für die Antwort @JoshuaWolff! Hilft dir das weiter @julian-weide?

**Comment by julian-weide** [comment]
Ja, danke @JoshuaWolff! Die Antwort hat mir in meinem Verständnis sehr weitergeholfen. Ich habe mich aber entschieden, kein Child-Theme einzurichten, da ich die Seite so einfach halten will, dass sie danach auch ohne mich verwaltet werden kann. Daher hoffe ich, dass in einer der nächsten Versionen die 404.php aktualisiert wird.
